internal/grpc/services/gateway/usershareprovider.go

```diff
@@ -698,10 +698,19 @@ func (s *svc) removeShare(ctx context.Context, req *collaboration.RemoveShareReq
 
 func (s *svc) removeSpaceShare(ctx context.Context, ref *provider.ResourceId, grantee *provider.Grantee) (*collaboration.RemoveShareResponse, error) {
 	listGrantRes, err := s.listGrants(ctx, ref)
+	var permissions *provider.ResourcePermissions
```

Comment:
can you move this line down? It doesn't make sense to have `permissions` created here before the `err` from `listGrants` is checked...

Reply:
done